Coordinator/Coordinator/Coordinator/BaseCoordinator.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  BaseCoordinator.swift
+//  Coordinator
+//
+//  Created by Vladka on 28.12.2022.
+//
+
+import UIKit
+
+/// Базовый координатор
+class BaseCoordinator {
+    
+    // MARK: - Public Properties
+    var childCoordinators: [BaseCoordinator] = []
+
+    // MARK: - Public Mhetods
+    func start() {}
```

Comment:
```suggestion
    func start() {}
    
```
где-то есть пробелы после марки где-то нету
лучше сделать в одном стиле